pulpcore/cli/ansible/repository.py

```diff
@@ -176,7 +176,7 @@ def repository(ctx: click.Context, pulp_ctx: PulpCLIContext, repo_type: str) ->
 repository.add_command(destroy_command(decorators=lookup_options))
 repository.add_command(task_command(decorators=nested_lookup_options))
 repository.add_command(version_command())
-repository.add_command(label_command())
+repository.add_command(label_command(decorators=lookup_options))
```

Comment:
should be:
```suggestion
repository.add_command(label_command(decorators=nested_lookup_options))
```
...until we removed the _old_ lookup options completely.